### PR TITLE
Group producer details

### PR DESCRIPTION
**Pull request by enricostano**

### CODE REVIEW

File: src/containers/Group/Producers/Item.js [hunk 1]
```diff
@@ -16,6 +17,7 @@ class Item extends Component {
 
     return (
       <div>
+        <Link to={`/groups/1/producers/${producer.id}`}>hola</Link>
```
Comment (enricostano) [inline src/containers/Group/Producers/Item.js]: Which should be the correct way? `./producers/${producer.id}` ?

Would be nice just `to={producer.id}` but it doesn't work

Comment (andresgutgon) [reply src/containers/Group/Producers/Item.js]: I think is ok being explicit:

``` javascript
`/groups/${group.id}/producers/${producer.id}`
```

Comment (enricostano) [reply src/containers/Group/Producers/Item.js]: yep but it would be nice to relative links working since you'll need to pass the `group` to all the childrens just to build the url in the links.

Anyway, let's try to get it working first, since it doesn't work even with this absolute hardcoded path 

```
to={`/groups/1/producers/${producer.id}\`}
```

File: src/containers/Group/Producers/CreateProducerModal/index.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { Modal } from 'react-bootstrap';
 
-import CreateProducerForm, { CREATE_PRODUCER_FORM } from 'components/forms/Producers/Create';
+import CreateProducerForm, { CREATE_PRODUCER_FORM } from 'components/forms/producers/Create';
```
Comment (andresgutgon) [inline src/containers/Group/Producers/CreateProducerModal/index.js:5]: Why this change?

**Review by andresgutgon — CHANGES_REQUESTED** [review]

**Review by andresgutgon — APPROVED** [review]
